pkg/manifest/manifest.go

```diff
@@ -346,7 +346,7 @@ func (pd *ParameterDefinition) Validate() error {
 
 	// Validate the Parameter Definition schema itself
 	if _, err := pdCopy.Schema.ValidateSchema(); err != nil {
-		return multierror.Append(result, err)
+		return multierror.Append(result, errors.Wrap(err, "encountered an error while validating parameter definition schema"))
```

Comment:
Oops, sorry I should have been more clear in my first review that the error should tell you which parameter it is, just like we do when validating the default value.

Reply:
Ah, no, my fault; was not paying careful enough attention.  Definitely makes sense to include the param/output name in the error message :)